modules/nf-core/ampcombi/main.nf

```diff
@@ -0,0 +1,48 @@
+process AMPCOMBI {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda (params.enable_conda ? "bioconda::ampcombi=0.1.5" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/ampcombi:0.1.5--pyhdfd78af_0':
+        'quay.io/biocontainers/ampcombi:0.1.5--pyhdfd78af_0' }"
+
+    input:
```

Comment:
missing `amp_database`

Reply:
The amp_database download is part of AMPcombi.

Reply:
The modules should also work in environments with no Internet. There should always be an option to supply a predownloaded database.

Reply:
The module indeed has an optional argument for the user to supply their own database indeed using the flag --amp-database instead of downloading it

Reply:
This needs to be an input so Nextflow can stage it properly. Optional inputs are handled by using an empty list when the input should be empty.
```nextflow
process TOOL {
    input:
    tuple val(meta), path(bam)
    path( opt_db )

    script:
    def db = optdb? "--db $opt_db": ""
    """
    cmd $db $bam
    """
}

workflow {
    TOOL (
        ch_bams,
        params.db ? file( params.db ) : []
    )
}
```

Reply:
Alright, we included the amp_database as an optional input, but still waiting for the conda version to bump our conda env to v0.1.6